com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/database/JDBCConnectEditView.java

```diff
@@ -114,8 +114,25 @@ private void addDriver() {
 		final String firstFile = fileDialog.open();
 		if (StringUtils.isBlank(firstFile)) {
 			return;
-		}
+		}		
 		if (!JDBCDriverManager.getInstance().addDriver(firstFile, false)) {
+			// --> added by chris
+			List<JDBCData> driverList = dt.getJDBCDatas();
+			JDBCData jd3 = dt.getJDBCData(firstFile) ;
+			if (jd3 != null)
+			{
```

Comment:
We are following Oracle's Java coding convention. If possible, please keep the same coding convention as other source code. Please see [here](https://www.oracle.com/technetwork/articles/javase/codeconvtoc-136057.html) for details.